packages/calcite-components/src/components/action-menu/action-menu.stories.ts

```diff
@@ -20,6 +20,24 @@ export const simple = (): string =>
     </calcite-action-menu>
   `;
 
+export const simpleSolid_TestOnly = (): string =>
+  html`<div style="background-color:#99ccff">
+    <calcite-action-menu>
+      <calcite-action text="Plus" icon="plus" text-enabled></calcite-action>
+      <calcite-action text="Minus" icon="minus" text-enabled></calcite-action>
+      <calcite-action text="Table" icon="table" text-enabled></calcite-action>
+    </calcite-action-menu>
+  </div> `;
```

Comment:
Does the background color need to be `#99ccff` for this test? If not the `simple` one above should cover the solid use case so we should be able to remove this one.

Reply:
Woah, TIL GitHub renders a little color icon next to color codes

Reply:
Yeah the background needs to be set to something to test that the transparent appearance is working correctly. I can switch it to just `red` to be a little more obvious.